ciphey/basemods/Decoders/URL.py

```diff
@@ -0,0 +1,40 @@
+from typing import Optional, Dict, List
+from urllib.parse import unquote
+
+from ciphey.iface import ParamSpec, Config, T, U, Decoder, registry
+
+
+@registry.register_multi((str, str), (bytes, bytes))
+class URL(Decoder[T, U]):
+    def decode(self, ctext: T) -> Optional[U]:
+        """Write the code that decodes here
+        ctext -> the input to the function
+        returns string
+    """
+        try:
+            ctext = ctext.replace("+", " ")  # replace + sign with spacebar for unquote
+            return unquote(ctext)  #built-in function that decodes URL coding
```

Comment:
Why is this a try / except, unquote does not raise an exception (at least in my testing) :) 

This means that every single thing can be decoded by this.

Try this:

```python
ctext = ctext.replace("+", " ")
result = unquote(ctext)
return result != ctext
```

Reply:
Ye, the function doesn't really raise exception :)
shouldn't the function return the decoded url?

Reply:
Ah yes you're right, it should be this:

```python
ctext = ctext.replace("+", " ")
result = unquote(ctext)
return result if result != ctext else None
```

It returns the decoded URL if it did something to it, and if not it returns None

Reply:
Also, do you mind putting the comments on new lines, and the 2nd comment should have a space between the hashtag and the comment itself `# like this`? It's for PEP8 :D